src/compiler/utilities.ts

```diff
@@ -1345,10 +1345,21 @@ namespace ts {
             && node.parent.parent.kind === SyntaxKind.VariableStatement;
     }
 
-    export function isValidESSymbolDeclaration(node: Node): node is VariableDeclaration | PropertyDeclaration | SignatureDeclaration {
-        return isVariableDeclaration(node) ? isVarConst(node) && isIdentifier(node.name) && isVariableDeclarationInVariableStatement(node) :
+    export function isCommonJsExportedExpresion(node: Node) {
+        if (!isInJSFile(node)) return false;
+        return (isObjectLiteralExpression(node.parent) && isBinaryExpression(node.parent.parent) && getAssignmentDeclarationKind(node.parent.parent) === AssignmentDeclarationKind.ModuleExports) ||

```

Comment:
What is this intended to be called on? Property assignments inside an object literal? I guess this is so we don't widen

```js
module.exports = {
  a: 1
}
```

It doesn't apply to 

```js
exports.x = {
  a: 1
}
```

does it?

Reply:
The second is an `ExportsProperty` assignment, not a `ModuleExports` assignment, so should remain unaffected.

Reply:
what about

```js
module.exports.x = {
  a: 1
}
```